velox/expression/tests/FunctionCallToSpecialFormTest.cpp

```diff
@@ -61,7 +61,9 @@ TEST_F(FunctionCallToSpecialFormTest, andCall) {
 TEST_F(FunctionCallToSpecialFormTest, castCall) {
   ASSERT_TRUE(isFunctionCallToSpecialFormRegistered("cast"));
 
-  ASSERT_THROW(resolveTypeForSpecialForm("cast", {}), VeloxRuntimeError);
+  ASSERT_THROW(
+      resolveTypeForSpecialForm("cast", std::vector<TypePtr>({})),
```

Comment:
why do we need this change?

Reply:
Because we introduce new API https://github.com/facebookincubator/velox/pull/5597#pullrequestreview-1616343200, cannot infer `{}` input type. Received compiler failure.